terracumber/terraformer.py

```diff
@@ -113,6 +113,17 @@ def get_hostname(self, resource):
                     return value[resource]['hostname']
         return None
 
+    def get_single_node_ipaddr(self):
+        """Get the hostname for a single node from tfstate file"""
+        with open(self.terraform_path + '/terraform.tfstate', 'r') as tf_state:
+            j = load(tf_state)
+            # This seems to be sumaform specific. I wonder if there is
```

Comment:
I am confused by this comment. If this is specific to sumaform, why is this function being used only for the salt shacker at https://github.com/uyuni-project/terracumber/pull/29/files#diff-56646f4f9cb011c599f92d8cd92395d82e90023b111c80931c0dcf8d8267bf47R217?

Reply:
Actually, this just a copy/paste from the similar function I used as template: https://github.com/uyuni-project/terracumber/pull/29/files#diff-8f0f56b3ac9934bb3212a28b2d2d2eab4e2fc0164d379434c576b1b0e4794e4bR106

IIUC what this message means is that, the parsing we are doing requires that sumaform `main.tf` file contains an specific `output` section. Since this is "terraformer" class, maybe the initial idea was to abstract it as much as possible from sumaform, so probably that this comment was made.

I hope this clarifies your question!

Reply:
Ah, I see now, you reused `get_hostname` and probably this comment was added by myself.

Right. Closing the thread.